package.json

```diff
@@ -25,6 +25,7 @@
     "promise-pjs": "1.1.4",
     "prop-types": "15.7.2",
     "react-dates": "15.5.3",
+    "renovate": "^19.239.8",
```

Comment:
Use exact version here (remove caret).

Reply:
Removed it entirely; will add as devDependency in the PR introducing the presubmit check